kbase-extension/static/kbase/js/widgets/appWidgets2/view/newObjectView.js

```diff
@@ -1,57 +1,42 @@
-define([
-    'bluebird',
-    'kb_common/html',
-    '../validation',
-    'common/events',
-    'common/runtime',
-    'common/dom',
-    'bootstrap',
-    'css!font-awesome',
-], (Promise, html, Validation, Events, Runtime, Dom) => {
+define(['bluebird', 'kb_common/html', 'common/dom', 'bootstrap', 'css!font-awesome'], (
+    Promise,
+    html,
+    Dom
```

Comment:
Fairly sure that this `Dom` module replicates (some of) the contents of the UI module.

Then again, if you've got some cool code, you don't want to restrict it to just a single file -- copy it elsewhere!!

Reply:
Yeah, I got a little sloppy with `newObjectView` and just did linting stuff. I'll clean it up a little further.